README.md

```diff
@@ -193,29 +192,29 @@ Client-side javascript testing is run through Jest. Run `npm run test`* from the
 
 #### Test Coverage
 
-Coverage is sent to codecov on commits to the repo (see bitbucket pipeline for branch to see branch coverage). Ideally we only merge positive code coverage changes to master.
+Coverage is sent to codecov on commits to the repo (see github actions for branch to see branch coverage). Ideally we only merge positive code coverage changes to master.
 
 
 #### Production Deployment
 
 The Core Portal runs in a Docker container as part of a set of services managed with Docker Compose.
 
-Portal images are built by [Jenkins](https://jenkins01.tacc.utexas.edu/view/Frontera%20Web/job/Frontera_Portal/) and published to the [Docker Hub repo](https://hub.docker.com/repository/docker/taccwma/frontera-portal).
+Portal images are built by [Jenkins](https://jenkins01.tacc.utexas.edu/view/WMA%20CEP/job/Core_Portal_Build/) and published to the [Docker Hub repo](https://hub.docker.com/repository/docker/taccwma/core-portal).
```

Comment:
__Suggestion.__ Changing all these redundant URLs is cumbersome, and diffs could conflict with any unrelated wording changes. I propose (now or later) using Markdown's link aliases. Example application: https://github.com/TACC/Core-Project-Resources/pull/1/commits/274573649cdb7dbd178c98b7035941d4e8201f1a